bolt-simple-taproot.md

```diff
@@ -0,0 +1,1489 @@
+
+  - MUST set the original, non-TLV `signature` field to a 0-byte-array of length
+    64.
+  
+  - MUST sort the exchanged `funding_pubkey`s using the `KeySort` algorithm from
+    `bip-musig2`.
+  
+  - MUST compute the aggregated `musig2` public key from the sorted
+    `funding_pubkey`s using the `KeyAgg` algorithm from `bip-musig2`.
+  
+  - MUST generate a unique nonce to combine with the `next_local_nonce`
+    previously received in `open_channel` using `NonceAgg` from `bip-musig2`.
+  
+  - MUST use the generated secret nonce and the calculated aggregate nonce to
+    construct a `musig2` partial signature for the sender's remote commitment
+    using the `Sign` algorithm from `bip-musig2`.
+  
+  - MUST include the partial signature and the public counterpart of the
+    generated nonce in the `partial_signature_with_nonce` field.
+
+The recipient:
+
+  - MUST fail the channel if `partial_signature_with_nonce` is absent.
+  
+  - MUST compute the aggregate nonce from:
+  
+    - the `next_local_nonce` field the recipient previously sent in
+      `open_channel`
+  
+    - the `public_nonce` included as part of the `partial_signature_with_nonce`
+      field
+  
+  - MUST verify the `partial_signature_with_nonce` field using the
+    `PartialSigVerifyInternal` algorithm of `bip-musig2`:
+  
+    - if the partial signature is invalid, MUST fail the channel
+
+#### `channel_ready` Extensions
+
+We add a new TLV field to the `channel_ready` message:
+
+1. `tlv_stream`: `channel_ready_tlvs`
+2. types:
+   1. type: 4 (`next_local_nonce`)
+   2. data:
+      * [`66*byte`: `public_nonce`]
+
+Similar to the `next_per_commitment_point`, by sending the `next_local_nonce`
+value in this message, we ensure that the remote party has our public nonce
+which is required to generate a new commitment signature.
+
+##### Requirements
+
+The sender:
+
+- MUST set `next_local_nonce` to a fresh, unique `musig2` nonce as specified by
+  `bip-musig2`
+
+The recipient:
+
+- MUST fail the channel if `next_local_nonce` is absent.
+
+### Cooperative Closure
+
+Compared to the base segwit v0 channel type, for simple taproot channels, the
+co-op close transaction now always signals RBF. In other words, the sequence
+field of the sole input to the cooperative close transaction MUST be
+less-than-or-equal to `0xfffffffd`. This enables a future cooperative closure
+flow to support increasing the fee of subsequent close offers via RBF.
+
+In addition, rather than adopt the existing cooperative closure fee rate
+"negotiation", the responder SHOULD now always accept the offer sent by the
+initiator. In other words, the cooperative close process now terminates after
+exactly 1 RTTs: initiator sends sigs with offer, with the responder echo'ing
+back the same fee rate. This serves to ensure that the co-op close process
+always terminates deterministically, and also plays nicer with the nonces: only
+a single message is ever signed by both sides for a coop close workflow.
```

Comment:
Looks like I missed that paragraph in my commits, which refers to the legacy closing protocol. Now that we updated this spec to rely on `option_simple_close` whenever using official taproot channels, we should remove this paragraph?

It looks like we can actually merge this section and the "Channel Cooperative Close Extensions" below into a single section and remove references to legacy vs modern close entirely?